recipes/flytekit/LICENSE

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
This license is also included in the tar ball.

Reply:
Gotcha. Removed the file.